vendor/plugins/edu.harvard.dce.paella.captionsPlugin/captionsPlugin.js

```diff
@@ -398,6 +402,22 @@ Class ("paella.plugins.DceCaptionsPlugin", paella.ButtonPlugin,{
 	        		paella.player.videoContainer.seekToTime(parseInt(secBegin));
 	        });
     	});
+    },
+
+    _addTagHeader: function(container, tags) {
+      var self = this;
+      if (!tags) return;
+      if ( ((Array.isArray && Array.isArray(tags)) || (tags instanceof Array)) == false) {
```

Comment:
I still don't follow the logic and may possibly be missing the point. when i see code like this i assume that the `tags` argument can be a string or an array, and if it's the former, then you want to change it to a array?